### PR TITLE
Update ACM COVID Policies page

### PR DESCRIPTION
**Pull request by alireza-jahandoost**
Change everything specified in [issue #479](https://github.com/uclaacm/website/issues/479)

### CODE REVIEW

File: pages/covid.js [hunk 1]
```diff
@@ -46,14 +50,14 @@ function COVID() {
...
-          <a className="button" href="/files/acm-covid-policies-s22-last-updated-03-27-22.pdf" target="_blank" rel="noopener noreferrer">
+          <a className="button" href="/files/acm-covid-policies-s22-last-updated-04-11-22.pdf" target="_blank" rel="noopener noreferrer">
             <FontAwesomeIcon icon={faFilePdf} fixedWidth aria-hidden={true} /> Link to PDF
           </a>
         </div>
         <br />
         <iframe
           title="ACM COVID Policies Spring 2022"
-          src="https://docs.google.com/document/d/e/2PACX-1vQIXLd4Omqv2XaDflpIlbmyfYyXC0GhE64XbS1kakNOHf-9PauJnIBTiNFz_rEQteixMecfN8fKld5v/pub?embedded=true"
+          src="https://docs.google.com/document/d/1IcBZ4yj4kJ14Gg0PVw3xbg-QqRYUQGUlHm8vVskgQOQ/edit?usp=sharing"
```
Comment (matthewcn56) [inline pages/covid.js:60]: For the embedded iframe, you have to use a special link made by publishing a google docs page to the web, here's the correct link! https://docs.google.com/document/d/e/2PACX-1vQIXLd4Omqv2XaDflpIlbmyfYyXC0GhE64XbS1kakNOHf-9PauJnIBTiNFz_rEQteixMecfN8fKld5v/pub?embedded=true

File: pages/covid.js [hunk 2]
```diff
@@ -29,11 +29,15 @@ function COVID() {
...
         <p>
-          Last major update: <b>Sunday, March 27th, 2022</b>.
+          Last major update: <b>Monday, April 11th, 2022</b>.
           <div>
             <p className='mb-0'>Version history</p>
             <ul className='mt-0'>
-              <li><b>Sunday, March 27th, 2022</b>: updated to reflect <a href='https://covid-19.ucla.edu/ucla-covid-protocols/' target="_blank" rel="noopener noreferrer"> UCLA&apos;s COVID-19 protocol updates for spring quarter</a></li>
+              <li>
+                  <b>Monday, April 11th, 2022</b>
+                  : Updated based on UCLA&apos;s Updates on COVID-19 Protocols for Spring Quarter
```
Comment (matthewcn56) [inline pages/covid.js:38]: Can you please embed a link into the `UCLA's Updates on COVID-19 Protocols for Spring Quarter` to point to UCLA's covid guidelines like done in the other bullet points? https://covid-19.ucla.edu/update-on-covid-19-protocols-for-spring-quarter/

**Review by matthewcn56 — CHANGES_REQUESTED** [review]
Thanks for taking a stab at this! There's just a couple small things I left in like the proper link for the iframe and adding in a hyperlink to ucla's covid updates, but other than that it looks great!

**Comment by alireza-jahandoost** [comment]
@matthewcn56 I've done the change requests. please check again.

**Review by matthewcn56 — APPROVED** [review]
Looks great, thanks for doing this!

**Pull request merged**